router/dataplane.go

```diff
@@ -145,12 +145,21 @@ type BatchConn interface {
 	// @ ensures   err == nil ==>
 	// @ 	forall i int :: { &msgs[i] } 0 <= i && i < n ==>
 	// @ 		MsgToAbsVal(&msgs[i], ingressID) == old(MultiReadBioIO_val(place, n)[i])
+	// SIF: classification spec
+	// NOTE: The postcondition for recv specifies that received messages are low.
+	// TODO: I have decided to mark the abstraction as low for now.
+	// And also used MultiReadBioIOI_val instead of MsgToAbsVal to match what is
+	// used in permissions.
```

Comment:
```suggestion
	// Classification spec for SIF.
	// The received messages are low, i.e., observable by the attacker.
	// TODO: we mark the abstraction as low for now and used MultiReadBioIOI_val
	// instead of MsgToAbsVal to match what is used in permissions.
```

Reply:
it is not clear how one is supposed to address this TODO